src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Гаджеты</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Планшеты</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Фото</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
according to style guide you shouldn't nest blocks in BEM. [Reference](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en). I would do somehitng like that:
```
<nav class="navigation">
    <ul class="navigation__list">
        <li class="navigation__item">
            <a href="#" class="navigation__link is-active">Apple</a>
        </li>
        <li class="navigation__item">
            <a href="#" class="navigation__link">Samsung</a>
        </li>
        <li class="navigation__item">
            <a href="#" class="navigation__link">Смартфоны и телефоны</a>
        </li>
        <li class="navigation__item">
            <a href="#" class="navigation__link" data-qa="nav-hover">Ноутбуки и компьютеры</a>
        </li>
        <li class="navigation__item">
            <a href="#" class="navigation__link">Гаджеты</a>
        </li>
        <li class="navigation__item">
            <a href="#" class="navigation__link">Планшеты</a>
        </li>
        <li class="navigation__item">
            <a href="#" class="navigation__link">Фото</a>
        </li>
        <li class="navigation__item">
            <a href="#" class="navigation__link">Видео</a>
        </li>
    </ul>
</nav>

```